packages/wrangler/src/dev/miniflare.ts

```diff
@@ -931,6 +931,8 @@ function getAssetServerWorker(
 					port: 0,
 				},
 			],
+			assetsPath: config.experimentalAssets.directory,
+			assetsKVBindingName: "ASSETS_KV_NAMESPACE",
```

Comment:
Should we conifigure `assetsManifestBindingName` here too?

Reply:
I left it out on purpose as ASW2 wasn't doing anything with, but I see you left a comment asking to add some basic handling of that binding in ASW. Will do that

Reply:
✅